libc-test/build.rs

```diff
@@ -1609,12 +1641,28 @@ fn test_freebsd(target: &str) {
         }
     });
 
+    cfg.skip_struct(move |ty| {
+        match ty {
+            // `mmsghdr` is not available in FreeBSD 10
+            "mmsghdr" if Some(10) == freebsd_ver => true,
+
+            _ => false,
+        }
+    });
+
     cfg.skip_fn(move |name| {
         // skip those that are manually verified
         match name {
             // FIXME: https://github.com/rust-lang/libc/issues/1272
             "execv" | "execve" | "execvp" | "execvpe" | "fexecve" => true,
 
+            // These functions were added in FreeBSD 11:
+            "fdatasync" | "aio_waitcomplete" | "mq_getfd_np"
```

Comment:
Actually, `aio_waitcomplete` has been present since FreeBSD 4.  But the return type changed in 11.0 (SVN r297167).

Reply:
Thanks, fixed.